### PR TITLE
ko update

### PR DESCRIPTION
**Pull request by Terrorboy**

### CODE REVIEW

**Comment by Terrorboy** [comment]
https://github.com/codeigniter4/translations/issues/148

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ claims no expertise beyond English.
 | id     | Indonesian           | 4.0.0   | [ridho1991][id]      | Mutasim Ridlo
 | it     | Italian              | 4.0.0   | [luk3b8][it]         | Luca Rasia
 | ja     | Japanese             | 4.0.3   | [kenjis][ja]         | Kenji Suzuki
-| ko     | Korean               | 4.0.2   | [codeigniter-kr][kr] | codeigniter-kr
+| ko     | Korean               | 4.0.4   | [codeigniter-kr][kr] | codeigniter-kr
```
Comment (paulbalandan) [inline README.md:45]: Please revert this change. `ko` was introduced in `v4.0.2`

**Review by paulbalandan — APPROVED** [review]
